tools/onnx-optimize.py

```diff
@@ -28,23 +29,32 @@ def get_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--input", required=True, help="onnx input model file")
     parser.add_argument("--output", help="output model file")
+    target_options = [constants.TARGET_CHANNELS_LAST, constants.TARGET_CHANNELS_FIRST]
+    parser.add_argument("--target", default=",".join(constants.DEFAULT_TARGET), choices=target_options,
+                        help="target platform")
     args = parser.parse_args()
+    args.target = args.target.split(",")
     return args
 
 
-def load_graph(fname):
+def load_graph(fname, target):
     model_proto = onnx.ModelProto()
     with open(fname, "rb") as f:
         data = f.read()
         model_proto.ParseFromString(data)
-    g = GraphUtil.create_graph_from_onnx_model(model_proto)
+    g = GraphUtil.create_graph_from_onnx_model(model_proto, target)
     return g, model_proto
 
 
 def main():
     args = get_args()
 
-    g, org_model_proto = load_graph(args.input)
+    g, org_model_proto = load_graph(args.input, args.target)
```

Comment:
Maybe it would be worth adding an example in the documentation? Is it available through all APIs?

Reply:
I think we could but right now onnx does not support this so it would be a little iffy.